core/include/traccc/edm/cell.hpp

```diff
@@ -123,9 +123,22 @@ using host_cell_container = host_container<cell_module, cell>;
 /// Convenience declaration for the cell container type to use in device code
 using device_cell_container = device_container<cell_module, cell>;
 
+/// Convenience declaration for the cell container type to use in device code
+/// (const)
+using device_cell_const_container =
+    device_container<const cell_module, const cell>;
+
 /// Convenience declaration for the cell container data type to use in host code
 using cell_container_data = container_data<cell_module, cell>;
 
+/// Convenience declaration for the cell container data type to use in host code
+/// (const)
+using cell_const_container_data = container_data<const cell_module, const cell>;
+
+/// Convenience declaration for the cell container view type to use in host code
+/// (const)
+using cell_const_container_view = container_view<const cell_module, const cell>;
```

Comment:
Let's call these `cell_container_const_data` and `cell_container_const_view` instead. Just to keep with the naming used for the spacepoint types.

Reply:
done